test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/Materialization/CollectionValueMaterializationPolicyTests.cs

```diff
@@ -141,7 +141,7 @@ public void DataServicCollectionOfTAsCollectionTypeShouldFailForPrimitiveOrCompl
         [Fact]
         public void CreateCollectionInstanceShouldFailOnTypeWithNoParametersLessConstructors()
         {
-            var testContext = new TestMaterializerContext();
+            var testContext = new TestMaterializerContext(new MaterializerCache());

```

Comment:
Would it be easier to update the default constructor for `TestMaterializerContext` so that it uses a new `MaterializerCache`?

Reply:
Yes it would be easier. I did think of that, but only after writing the tests and realizing just how many changes I made as a result. I can refactor it again.